dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -37,13 +37,15 @@ internal class RumSessionScope(
     frameRateVitalMonitor: VitalMonitor,
     internal val sessionListener: RumSessionListener?,
     contextProvider: ContextProvider,
+    applicationDisplayed: Boolean,
     appStartTimeProvider: AppStartTimeProvider = DefaultAppStartTimeProvider(),
     private val sessionInactivityNanos: Long = DEFAULT_SESSION_INACTIVITY_NS,
     private val sessionMaxDurationNanos: Long = DEFAULT_SESSION_MAX_DURATION_NS
 ) : RumScope {
 
     internal var sessionId = RumContext.NULL_UUID
     internal var sessionState: State = State.NOT_TRACKED
+    internal var isActive: Boolean = true
```

Comment:
should be `false` by default to be aligned with `RumContext.NULL_UUID` for `sessionId` imo.

Reply:
It appears there are actually a lot of tests that assume a session is active by default, even without a SessionId. (`activeSession` on `RumApplicationScope` just returned the session it had.)

I'm curious why we don't go the opposite direction to make it consistent, and have the session generate its id on creation?